pages/common/ghc.md

```diff
@@ -0,0 +1,23 @@
+# ghc
+
+> The Glassgow Haskell Compiler, compiles and links Haskell source files.
```

Comment:
maybe use a period, dash, semicolon or colon (your preference) instead of a comma here.